.ruff-excludes.toml

```diff
@@ -1156,6 +1156,9 @@
     "I001",     # https://docs.astral.sh/ruff/rules/unsorted-imports
     "UP008",    # https://docs.astral.sh/ruff/rules/super-call-with-parameters
 ]
+"./scripts/west_commands/patch.py" = [
+    "I001",     # https://docs.astral.sh/ruff/rules/unsorted-imports
+]
```

Comment:
I formatted the file with ruff and it's still complaining that it isn't formatted according to ruff, so 🤷🏼‍♂️

Reply:
For some reason, ruff does not seem to sort these properly at all on my machne, even with the latest version.

Reply:
Not sure if you read my other comment, make sure to remove this entry from the excludes file, otherwise it will be taken into account when running `ruff check`.